src/prefect/server/schemas/schedules.py

```diff
@@ -137,51 +163,138 @@ def _get_dates_generator(
             else:
                 n = 1
 
-        if start is None:
-            start = now("UTC")
+        if sys.version_info >= (3, 13):
```

Comment:
There's a big version branch here to handle interval schedules with `whenever` for Python 3.13. The existing logic relied heavily on `pendulum` and `whenever` didn't mix well with it so a full split was cleaner.

Reply:
I might have made this two completely separate functions for clarity